tests/test_datasets.py

```diff
@@ -14,10 +13,8 @@ class TestDatasets(geomstats.tests.TestCase):
 
     def setUp(self):
         """Set up tests."""
-        self.g1 = Graph()
-        self.g2 = Graph(
-            graph_matrix_path=data_utils.KARATE_PATH,
-            labels_path=data_utils.KARATE_LABELS_PATH)
+        self.g1 = data_utils.load_random()
```

Comment:
Maybe transform `test_graph_load` into `test_load_karate` and `test_load_random_graph` to decouple the unit tests, and see if they pass in all backends?